algorithms/kernel/low_order_moments/oneapi/low_order_moments_online_oneapi_impl.i

```diff
@@ -232,13 +229,17 @@ LowOrderMomentsOnlineTaskOneAPI<algorithmFPType, scope>::LowOrderMomentsOnlineTa
 
     nColsBlocks = (nFeatures + maxWorkItemsPerGroup - 1) / maxWorkItemsPerGroup;
 
-    nRowsBlocks = 8;
+    nRowsBlocks = 128;
     if (nVectors < 5000)
         nRowsBlocks = 1;
     else if (nVectors < 10000)
-        nRowsBlocks = 2;
+        nRowsBlocks = 8;
     else if (nVectors < 20000)
-        nRowsBlocks = 4;
+        nRowsBlocks = 16;
+    else if (nVectors < 50000)
+        nRowsBlocks = 32;
+    else if (nVectors < 100000)
+        nRowsBlocks = 64;
 
     workItemsPerGroup = (maxWorkItemsPerGroup < nFeatures) ? maxWorkItemsPerGroup : nFeatures;
```

Comment:
I see two equivalent blocks of code in batch and online version. I think, we need to figure out how to unify this code for batch and online modes. I'm not asking you to do it now, but this technical debt shall be addressed with the transition to new interfaces.

Reply:
Agree, will address it during transition to new interfaces.